arm64: dts: mediatek: mt7988: add trng node

Add support for trng on mt7988a.

Tested on Banana Pi BPI-R4.

Signed-off-by: Marcos Alano <marcoshalano@gmail.com>
---
--- a/arch/arm64/boot/dts/mediatek/mt7988a.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt7988a.dtsi
@@ -1319,4 +1319,8 @@
 			     <GIC_PPI 11 IRQ_TYPE_LEVEL_LOW>,
 			     <GIC_PPI 10 IRQ_TYPE_LEVEL_LOW>;
 	};
+
+	trng {
+		compatible = "mediatek,mt7988-rng";
+	};
 };
